git-cliff/src/args.rs

```diff
@@ -193,6 +193,9 @@ pub struct Opt {
 	/// Bumps the version for unreleased changes.
 	#[arg(long, help_heading = Some("FLAGS"))]
 	pub bump:             bool,
+	/// Bumps the version for unreleased changes with specified change.
+	#[arg(long, value_name = "BUMP_AS")]
+	pub bump_as:          Option<String>,
```

Comment:
How about we merge this functionality with the `--bump` argument like so?

```sh
git-cliff --bump # bumps based on conventional commits
git-cliff --bump major # bumps major
git-cliff --bump minor # bumps minor
git-cliff --bump patch # bumps patch 
```

To do that we can make `--bump` take an optional value:

```rust
	/// Bumps the version for unreleased changes.
	#[arg(long)]
	pub bump:             Option<Option<String>>,
```

What do you think? I think it would be cleaner this way :)

Reply:
I made it `<Option<BumpOption>>` and I created a separate enum `BumpType` for the config.

For the value parser i simply copied the existing example. Can you double check on that part?